.github/workflows/toranj.yml

```diff
@@ -181,6 +181,31 @@ jobs:
       run: |
         ./tests/toranj/build.sh posix-15.4
 
+  nexus:
+    name: nexus
+    runs-on: ubuntu-20.04
```

Comment:
Can we use `ubuntu-22.04`?

Reply:
Changed it for this this job.
The rest of the jobs in this `yml` file use the `20.04` (may make sense to change them as well, in a seperate PR)